rockset/resource_alias.go

```diff
@@ -39,6 +40,13 @@ func resourceAlias() *schema.Resource {
 				Required:     true,
 				ValidateFunc: rocksetNameValidator,
 			},
+			"overwrite": {
+				Description: "Whether existing alias should be overwritten.",
+				Type:        schema.TypeString,
+				Default:     "false",
+				ForceNew:    false,
+				Optional:    true,
+			},
```

Comment:
This goes against the terraform model, as you want it to stop when it encounters an existing  resource for which it has no information, it will stop. If you want to manage an existing resource, you first have to import it so terraform "owns" it before it would be modified.

Just run `terraform import` and provide the resource path and the id (`qa.c` in your example)